Kernel/Devices/ConsoleDevice.cpp

```diff
@@ -58,7 +61,9 @@ void ConsoleDevice::put_char(char ch)
 {
     Kernel::SpinlockLocker lock(g_console_lock);
 #ifdef CONSOLE_OUT_TO_BOCHS_DEBUG_PORT
-    IO::out8(IO::BOCHS_DEBUG_PORT, ch);
+#    if ARCH(I386) || ARCH(X86_64)
+    bochs_debug_output(ch);
+#    endif
 #endif
```

Comment:
Can't we change this to `debug_output` which compiles and works on all architectures? The compile flag for Bochs can/should be moved into `bochs_debug_output` (as the flag is not checked in all debug output users IIRC).

Reply:
This will be fixed in https://github.com/SerenityOS/serenity/pull/15173.